.cirrus.yml

```diff
@@ -16,604 +8,578 @@ env:
     # Name of the ultimate destination branch for this CI run, PR or post-merge.
     DEST_BRANCH: "master"
     # Overrides default location (/tmp/cirrus) for repo clone
-    GOPATH: "/var/tmp/go"
+    GOPATH: &gopath "/var/tmp/go"
     GOBIN: "${GOPATH}/bin"
-    GOSRC: "/var/tmp/go/src/github.com/containers/podman"
-    CIRRUS_WORKING_DIR: "/var/tmp/go/src/github.com/containers/podman"
+    GOCACHE: "${GOPATH}/cache"
+    GOSRC: &gosrc "/var/tmp/go/src/github.com/containers/podman"
+    CIRRUS_WORKING_DIR: *gosrc
     # The default is 'sh' if unspecified
     CIRRUS_SHELL: "/bin/bash"
     # Save a little typing (path relative to $CIRRUS_WORKING_DIR)
     SCRIPT_BASE: "./contrib/cirrus"
-    # Command to prefix output lines with timing information
-    # (can't do inline awk script, Cirrus-CI or YAML mangles quoting)
-    TIMESTAMP: "awk --file ${CIRRUS_WORKING_DIR}/${SCRIPT_BASE}/timestamp.awk"
-
-    # HTMLify ginkgo and bats logs
-    LOGFORMAT: "${CIRRUS_WORKING_DIR}/${SCRIPT_BASE}/logformatter"
 
     ####
     #### Cache-image names to test with (double-quotes around names are critical)
-    ###
+    ####
     FEDORA_NAME: "fedora"
     PRIOR_FEDORA_NAME: "prior-fedora"
```

Comment:
Nooooooo! What happened? These were nice happy human-readable names until now, why are they changing to horrible un-numbered ones? Can this be reverted?

Reply:
maybe but damn, there was a good reason I had to change it and now I can't remember.  I'll try putting it back and see what breaks.